src/BaGet/Startup.cs

```diff
@@ -75,12 +80,15 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
             app.UseSpa(spa =>
             {
                 spa.Options.SourcePath = "../BaGet.UI";
-
+                var spaStaticFilesService = app.ApplicationServices.GetService<ISpaStaticFileProvider>();
+                spa.Options.DefaultPageStaticFileOptions = new StaticFileOptions();
+                spa.Options.DefaultPageStaticFileOptions.FileProvider = spaStaticFilesService.FileProvider;
                 if (env.IsDevelopment())
                 {
-                    spa.UseReactDevelopmentServer(npmScript: "start");
+                    //spa.UseReactDevelopmentServer(npmScript: "start");
```

Comment:
Do we need to support a custom host name while in development mode? That seems like a weird use case to me. Would it be better if we didn't support a custom host name while in development mode? The React development server is very useful :)

Reply:
Yeah you are right. I don't know enough about the react environment. But maybe it is possible to have like dev environment where homepage is set to `""` and a prodution environment where homepage is set to `__BAGET_PATH_BASE_PLACEHOLDER__`. So that npm (react) build setups the runtime and the index.html relative paths correctly.

I will try to find a appropiate solution for that.